catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplication.java

```diff
@@ -261,6 +278,29 @@ private void registerResources(CatalogApplicationConfig config, Environment envi
     environment.getApplicationContext().setErrorHandler(eph);
   }
 
+  private void intializeWebsockets(Environment environment) {
+    EngineIoServerOptions eioOptions = EngineIoServerOptions.newFromDefault();
+    eioOptions.setAllowedCorsOrigins(null);
+    WebSocketManager.WebSocketManagerBuilder.build(eioOptions);
+    environment.getApplicationContext().setContextPath("/");
+    if (socketAddressFilter != null)
+      environment
+          .getApplicationContext()
+          .addFilter(new FilterHolder(socketAddressFilter), "/socket.io/*", EnumSet.of(DispatcherType.REQUEST));
+    environment.getApplicationContext().addServlet(new ServletHolder(new FeedServlet()), "/socket.io/*");
+    // Upgrade connection to websocket from Http
+    try {
+      WebSocketUpgradeFilter webSocketUpgradeFilter =
+          WebSocketUpgradeFilter.configureContext(environment.getApplicationContext());
+      webSocketUpgradeFilter.addMapping(
+          new ServletPathSpec("/socket.io/*"),
```

Comment:
lets add a proper path here we can add /api/v1/push/feed

Reply:
done